playwright/package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "@types/jest": "^26.0.20",
+    "axios": "^0.21.1",
```

Comment:
could we use isomorphic-unfetch for normalisation? It just needs to be installed.